nautobot/core/templates/inc/media.html

```diff
@@ -83,6 +83,11 @@
             src: url('{% static 'fonts/UbuntuMono-Bold.woff2' %}') format('woff2');
             unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
         }
+        h1::before {
+            content: url("{% static 'img/nautobot_chevron_header.svg' %}");
+            position: relative;
+            top: 15px;
+        }
```

Comment:
Clever place to put this! Not sure it's a pattern we want to propagate though... it'd probably be better in the long term to have a templated css file to contain this and the font-faces above rather that sticking it all directly in `media.html`, maybe?